std/format.d

```diff
@@ -5419,31 +5420,6 @@ enum Mangle : char
     Timmutable = 'y',
 }
 
-// return the TypeInfo for a primitive type and null otherwise.  This
-// is required since for arrays of ints we only have the mangled char
-// to work from. If arrays always subclassed TypeInfo_Array this
-// routine could go away.
-private TypeInfo primitiveTypeInfo(Mangle m)
```

Comment:
So `primitiveTypeInfo` is a relict from the old `doFormat` days. It's not used anywhere anymore.
See e.g. https://github.com/dlang/phobos/pull/4978